utils/notebook_init_utils.py

```diff
@@ -0,0 +1,69 @@
+import pathlib
```

Comment:
Consider adding a docstring to the top of this file to help describe what it is for.